src/WebJobs.Extensions/Extensions/Timers/Scheduling/CronSchedule.cs

```diff
@@ -60,9 +47,10 @@ public override string ToString()
         internal static bool TryCreate(string cronExpression, out CronSchedule cronSchedule)
         {
             cronSchedule = null;
+            var isSixPart = cronExpression.Split(new[] { ' ' }, StringSplitOptions.RemoveEmptyEntries).Length == 6;
```

Comment:
A couple things:

Can we convert this to match 5 part cron expressions so that the default is 6 (in the event that the matching logic is off)

Also, I believe that the expressions support tabs and newlines in whitespace - can we change the match logic to be a regex with 5 parts and whitespace?

Reply:
Sure I'll update the PR